src/main/java/org/example/models/Project.java

```diff
@@ -0,0 +1,45 @@
+package org.example.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Project {
+
+    private String projectName;
+    private double projectValue;
+    private int clientID;
+
+    @JsonCreator
+    public Project(
+            @JsonProperty("projectName1") final String projectName1,
```

Comment:
Any reason why these are named 'somethingSomething1'

Reply:
some of the CheckStyle errors were telling me they were duplicate names and to rename them?